doc/generated/examples/caching_ex-random_1.xml

```diff
@@ -1,8 +1,8 @@
 <screen xmlns="http://www.scons.org/dbxsd/v1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.scons.org/dbxsd/v1.0 http://www.scons.org/dbxsd/v1.0/scons.xsd">% <userinput>scons -Q</userinput>
-cc -o f4.o -c f4.c
-cc -o f2.o -c f2.c
-cc -o f3.o -c f3.c
 cc -o f1.o -c f1.c
+cc -o f3.o -c f3.c
```

Comment:
I'd just drop the two generated examples from the changeset, as they add no value. On the other hand, they don't hurt anything, either.

Reply:
Hmm. that's just from running the doc tools. So that change is from a previous change which hadn't yet been regenerated by the doc tools.

Reply:
notice the word "random" in the name :)  you did update these for 4.4.0, so this isn't a missing change, it's just that it's different pretty much every run.

Reply:
ahh. yes.. :)